src/display/general/overallGraphs.go

```diff
@@ -30,10 +29,54 @@ import (
 	"github.com/pesos/grofer/src/utils"
 )
 
+var myPage *MainPage
 var isCPUSet = false
-
 var run = true
 var helpVisible = false
+var cpuTableVisible = false
+
+// Function runs when 3 tables are enabled in the Main page
+func tableChange_3(selectedTable int) {
+	switch selectedTable {
+	case 0:
+		myPage.CPUTable.BorderStyle.Fg = ui.ColorClear
+		myPage.DiskChart.BorderStyle.Fg = ui.ColorCyan
+		myPage.TemperatureTable.BorderStyle.Fg = ui.ColorCyan
+		myPage.DiskChart.ShowCursor = false
+		myPage.TemperatureTable.ShowCursor = false
+
+	case 1:
+		myPage.CPUTable.BorderStyle.Fg = ui.ColorCyan
+		myPage.DiskChart.BorderStyle.Fg = ui.ColorClear
+		myPage.TemperatureTable.BorderStyle.Fg = ui.ColorCyan
+		myPage.DiskChart.ShowCursor = true
+		myPage.TemperatureTable.ShowCursor = false
+
+	case 2:
+		myPage.CPUTable.BorderStyle.Fg = ui.ColorCyan
+		myPage.DiskChart.BorderStyle.Fg = ui.ColorCyan
+		myPage.TemperatureTable.BorderStyle.Fg = ui.ColorClear
+		myPage.DiskChart.ShowCursor = false
+		myPage.TemperatureTable.ShowCursor = true
+	}
+}
+
+//Function runs when 2 tables are enabled in the Main page
```

Comment:
```suggestion
// Function runs when 2 tables are enabled in the Main page
```
Comments (language agnostic) are usually `//` followed by a space

Reply:
Oh oops. Missed that lol